src/api/Intelligence.js

```diff
@@ -70,20 +69,13 @@ class Intelligence extends Base {
 
         const url = `${this.getBaseApiUrl()}/ai/extract_structured`;
 
-        let suggestionsResponse = {};
-        try {
-            suggestionsResponse = await this.xhr.post({
-                url,
-                data: request,
-                id: `file_${request.items[0].id}`,
-            });
-        } catch (e) {
-            const { status } = e;
-            if (isUserCorrectableError(status)) {
-                throw e;
-            }
-        }
-        return getProp(suggestionsResponse, 'data', {});
+        const suggestionsResponse = await this.xhr.post({
+            url,
+            data: request,
+            id: `file_${request.items[0].id}`,
```

Comment:
Do we need to make sure request has the `items` field and that it is not empty?

Reply:
I've added the same checks as are in `ask` method